### ISSUE TITLE
oAuthPrompt has an invalid value/link

### ISSUE THREAD

**Issue by darrenj** [Microsoft/BotFramework-Emulator#1006]
## Version
4.0.41704 and 4.0.55-one-off

## Describe the bug

When an OAuthPrompt is returned in Emulator Build 4.0.0-preview.40025 it has this attachment (and works)
```
{
  "attachments": [
    {
      "content": {
        "buttons": [
          {
            "text": "You haven't linked any accounts yet, if possible please authenticate your assistant to access this service or switch to your companion experience.",
            "title": "Skill Authentication",
            "type": "openUrl",
            "value": "oauthlink://https://token.botframework.com/api/oauth/signin?signin=REDACTEDIDENTIFIER-GUID|livechat"
          }
        ],
        "connectionName": "AzureAd",
        "text": "You haven't linked any accounts yet, if possible please authenticate your assistant to access this service or switch to your companion experience."
      },
      "contentType": "application/vnd.microsoft.card.oauth"
    }
  ],
```

However in 4.0.41704 and 4.0.55-one-off it returns this, you can see the moniker (oauth://) is incorrect versus oauthlink above and the URL is completely malformed.
```
{
  "attachments": [
    {
      "content": {
        "buttons": [
          {
            "text": "You haven't linked any accounts yet, if possible please authenticate your assistant to access this service or switch to your companion experience.",
            "title": "Skill Authentication",
            "type": "openUrl",
            "value": "oauth://AzureAd&&&REDACTEDGUIDE|livechat"
          }
        ],
        "connectionName": "AzureAd",
        "text": "You haven't linked any accounts yet, if possible please authenticate your assistant to access this service or switch to your companion experience."
      },
      "contentType": "application/vnd.microsoft.card.oauth"
    }
  ],
```

This then results in a malformed tokens/response event being sent to the Bot.

## To Reproduce
1. Use AuthenticationBot Sample and click on an oauth card
2. Observe malformed token being sent to bot
3. Prompt is shown again

## Expected behavior
OAuthPrompt to work and generate a correct token from ABS (as per previous builds)

## Screenshots
![image](https://user-images.githubusercontent.com/33519520/47212457-212abb00-d390-11e8-94cc-c9f96c941d10.png)

[bug]

**Comment by justinwilaby** [comment]
Fixed with #1008

**Comment by CKGrafico** [comment]
Same issue now with emulator `4.3.2` I've downloaded `4.1.0` and nothing I've downloaded `4.2.0` and nothing.
is working well outside the emulator

![image](https://user-images.githubusercontent.com/2048511/54203174-f8879700-44d1-11e9-9ab2-35625ff91c1a.png)
